__mocks__/@wordpress/api-fetch/index.js

```diff
@@ -0,0 +1,6 @@
+const apiFetch = jest.fn( () => {
+	return apiFetch.mockReturnValue;
+} );
+apiFetch.mockReturnValue = 'mock this value by overriding apiFetch.mockReturnValue';
```

Comment:
Maybe make this a full sentence? `'Mock this value by overriding apiFetch.mockReturnValue.'`

Reply:
I copied it over - can tweak 👍